examples/cifar10/cifar10/data_set.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+
+from keras.datasets import cifar10
+from keras.utils import to_categorical
+from sklearn.model_selection import train_test_split
+
+LABELS = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
+LABELS_NAMES = ['airplane', 'automobile', 'bird', 'cat', 'deer', 'dog', 'frog', 'horse', 'ship', 'truck']
+
+
+# Preprocess Function
+def preprocess_func():
+    (data_X, data_Y), (_, _) = cifar10.load_data()
```

Comment:
might be cleaner to do this if poosible:
(data_X, data_Y), _ = cifar10.load_data()